### DISCUSSION TITLE
`PISCES` validation

### DISCUSSION THREAD

**Discussion by jagoosw** [OceanBioME/OceanBioME.jl#226] [General]
I thought I should make a record of the current state of `PISCES` and what we think needs doing before we can consider it to be "working" so that it can be discussed.

As it stands, in the simple cases I tested (i.e. artificially forced box and column), the model seems to produce "reasonable" results and correctly conserves the element masses (C, Fe, P, Si, N), but we have no direct comparison to the other PISCES implementations.

We think that the best thing to do would be to have (a) simple case/cases in one of the other versions with as little physics as possible and check that we replicate it. 
When I started doing this I tried to use FABM since they have a box model, but I couldn't get it to compile with PISCES and then I got busy with other things.

**Comment by ali-ramadhan** [comment]
Just curious if any of the PISCES validations are publicly available online in papers or web pages?

**Reply by jagoosw** [reply]
I don't think they have any results except global ones, and a few plots in this tutorial https://zenodo.org/records/7139521 which I guess we might be able to use (but I do think we should check that all of the tracers are correct not just the ones they've plotted) but we would need to replicate their physics to some degree of accuracy.
